src/shared/api/genomeNexusClientInstance.ts

```diff
@@ -2,10 +2,12 @@ import { GenomeNexusAPI } from 'genome-nexus-ts-api-client';
 
 async function checkVersion(client: GenomeNexusAPI) {
     const versionResp = await client.fetchVersionGET({});
-    if (parseInt(versionResp.version.split('.')[0]) !== 1) {
+    if (
+        parseInt(versionResp.genomeNexus.database.version.split('.')[0]) !== 1
```

Comment:
```suggestion
        parseInt(versionResp.genomeNexus.server.version.split('.')[0]) !== 1
```
@averyniceday Maybe we should check `server` instead? Please correct me if wrong